_studio/mfx_lib/decode/av1/src/mfx_av1_dec_decode.cpp

```diff
@@ -1171,8 +1190,14 @@ mfxStatus VideoDECODEAV1::FillOutputSurface(mfxFrameSurface1** surf_out, mfxFram
     UMC::VideoDataInfo const* vi = fd->GetInfo();
     MFX_CHECK(vi, MFX_ERR_DEVICE_FAILED);
 
+    surface_out->Data.TimeStamp = bs->TimeStamp != static_cast<mfxU64>(MFX_TIMESTAMP_UNKNOWN) ? bs->TimeStamp : GetMfxTimeStamp(m_frame_order * m_in_framerate);
+    surface_out->Data.FrameOrder = m_frame_order;
+    surface_out->Data.Corrupted = 0;
+    surface_out->Info.PicStruct = MFX_PICSTRUCT_PROGRESSIVE;
     surface_out->Info.CropW = static_cast<mfxU16>(pFrame->GetUpscaledWidth());
-    surface_out->Info.CropH = static_cast<mfxU16>(pFrame->GetHeight());
+    surface_out->Info.CropH = static_cast<mfxU16>(pFrame->GetFrameHeight());
+    surface_out->Info.AspectRatioW = 1;
+    surface_out->Info.AspectRatioH = 1;
```

Comment:
There is still no:
- FrameRate info.
- mfxExtDecodedFrameInfo
 - proper handling of  Corrupted

Reply:
FrameRateInfo: I have updated in the latest commit.
mfxExtDecodedFrameInfo: I think we have done it in SetFrameType() which is called by :
https://github.com/Intel-Media-SDK/MediaSDK/blob/master/_studio/mfx_lib/decode/av1/src/mfx_av1_dec_decode.cpp#L126-L146
https://github.com/Intel-Media-SDK/MediaSDK/blob/master/_studio/mfx_lib/decode/av1/src/mfx_av1_dec_decode.cpp#L1167
Corrupted: IMHO, AV1 should have not special handling like VP9. @IlyaSergeyev Do you think so?

Reply:
Okay. Thanks. So only 'Corrupted' remained not covered here.

Reply:
Ahh, I just saw it's already covered here: https://github.com/Intel-Media-SDK/MediaSDK/blob/master/_studio/mfx_lib/decode/av1/src/mfx_av1_dec_decode.cpp#L674